src/extension.ts

```diff
@@ -10,7 +10,7 @@ import CommitProvider from './lib/editor/provider';
 
 export function activate(context: vscode.ExtensionContext) {
   output.initialize();
-  output.appendLine('Activated');
+  output.appendLine('[info] Extension Activated');
```

Comment:
We can refactor `output` to have `output.info`, `output.warning` and `output.error` APIs

Reply:
@vivaxy I will try to refactor it.